cms/djangoapps/contentstore/rest_api/urls.py

```diff
@@ -4,10 +4,12 @@
 
 from django.urls import include, re_path
 
+from .v0 import urls as v0_urls
```

Comment:
I've called it v0 because in this case, the API is just exposing an existing endpoint without too much thought put into the structure of the response etc. It's so we can get started with using this from MFEs quicker. I'd expect a v1 of the API as the next iteration that optimises how this works in a way that works better for MFEs. 

For instance, I'd expect this API to be split into multiple other APIs, one for each app that can be used to configure it, instead of one single API for all misc operations. 

On the other hand, if we can maintain this API as-is, I don't see any reason to not call it v1, and just do the next iteration as v2.